CONTRIBUTING.md

```diff
@@ -16,6 +16,8 @@ Remember that you can continue to push to your `feature/*` or `fix/*` branches t
 
 Please note that your pull request will have to pass the CI tests in order to be merged.
 
+Including unit tests is very welcomed and highly recommended. xpcc has its own simple and easy to use unit test framework. Unit tests are placed in the `test` subdirectory of every component (e.g. `xpcc/src/xpcc/math/filter/test`) and are run with `scons unittest`. See TESTING.md for more information about testing strategies in xpcc.
+
```

Comment:
This is too good to not put this into the README.md TL;DR section at the top!
Maybe something like this?
```md
[…]
- We have Continuous Integration as well.
- [And we care a lot about unit testing](TESTING.md).
- You have questions?
[…]
```